modules/acquia_cms_headless/tests/src/Functional/HeadlessFrontpageTest.php

```diff
@@ -2,29 +2,36 @@
 
 namespace Drupal\Tests\acquia_cms_headless\Functional;
 
+use Drupal\Tests\BrowserTestBase;
+
 /**
  * Tests for acquia_cms_headless frontpage.
  *
  * @group acquia_cms_headless
  * @group low_risk
  */
-class HeadlessFrontpageTest extends HeadlessTestBase {
+class HeadlessFrontpageTest extends BrowserTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'stark';
 
   /**
    * {@inheritdoc}
    */
   protected static $modules = [
+    'acquia_cms_headless',
     'acquia_cms_headless_ui',
   ];
 
   /**
    * Assert that frontpage for non logged-in user is login page.
    */
   public function testFrontPageIsLoginPage(): void {
-    $this->drupalLogout();
     $this->drupalGet('/frontpage');
-    $assert_session = $this->assertSession();
-    $assert_session->elementExists('css', '#user-login-form');
+    $this->assertSession()->elementExists('css', '#user-login-form');
+    $this->assertSession()->statusCodeEquals(200);
```

Comment:
@apathak18 no need to check for statusCodeEquals(200) as we are already checking for user-login-form ID which itself says that it is user login page

Reply:
@deepakmishra2 that's the element which we checked and have one more assertion which tells the page is returning 200 status.

Reply:
@apathak18 This doesn't make any sense , as you are not loading any page. It just checks for the user-login-form . Status codes are checked when you move from 1 link to another

Reply:
If you want to test for status code , it should be tested just after $this->drupalGet('/frontpage');